stdlib/asm/crypto/stark/verifier.masm

```diff
@@ -39,6 +39,10 @@ const.AUX_TRACE_COM_PTR=4294967289        # COMPOSITION_POLY_COM_PTR + 1
 const.MAIN_TRACE_COM_PTR=4294967290       # AUX_TRACE_COM_PTR + 1
 const.Z_PTR=4294967291                    # MAIN_TRACE_COM_PTR + 1
 const.LDE_SIZE_PTR=4294967292             # Z_PTR + 1
+const.C_PTR=4294967293                    # LDE_SIZE_PTR + 1
+const.R2_PTR=4294967294                   # C_PTR + 1
+const.R1_PTR=4294967295                   # R2_PTR + 1
```

Comment:
These declarations are probably not needed here.

Reply:
I left them as a comment, `TRACE_LENGTH_PTR` itself is also not necessary here. I can remove it, but without this is hard to figure out which addresses are unused (on my first implementation I used the first address that was not mentioned, i.e. 4294967293, which broke the implementation)

Reply:
I'd probably add a comment here that these are unused then - or something like that. We should probably refactor this to move shared constants into a separate file (e.g., `constants.masm` similar to [this](https://github.com/0xPolygonMiden/miden-base/blob/main/miden-lib/asm/sat/constants.masm)) or maybe even create a separate memory layout module (e.g., `mem_layout.mams` similar to [this](https://github.com/0xPolygonMiden/miden-base/blob/main/miden-lib/asm/sat/layout.masm)).

Reply:
added a comment